### ISSUE TITLE
Is it useful to use dcn on 1x1conv?

### ISSUE THREAD

**Issue by kirkzZ** [chengdazhi/Deformable-Convolution-V2-PyTorch#85]
hello,is it useful to use dcn on 1x1conv?

**Comment by dazhi-cheng** [comment]
I would expect less effect on 1x1conv, simply because 1x1 conv is not doing much feature aggregation. Deformable conv helps mostly with dynamic feature aggregation. But still it would be interesting to see the results.

**Comment by kirkzZ** [comment]
Yes, from my experimental results, there is only a little increase, but there is no comparison, so I don't know whether my result is normal

**Comment by dazhi-cheng** [comment]
Good to know!
